rllib/algorithms/ppo/torch/ppo_torch_rl_module.py

```diff
@@ -1,4 +1,4 @@
-from typing import Any, Dict
+from typing import Any, Dict, Mapping
```

Comment:
Can we remove `Mapping` (replace with `Dict`)? I find it confusing. In the end, we work with dicts anyways. Mapping is too generic and doesn't really match here. It's an artifact from earlier times :)